authors/apps/authentication/views.py

```diff
@@ -28,9 +31,31 @@ def post(self, request):
         serializer = self.serializer_class(data=user)
         serializer.is_valid(raise_exception=True)
         serializer.save()
-
+        email = serializer.data['email']
+        verification_token = RegistrationAPIView.verification_token(email)
+        RegistrationAPIView.verification_link(email,verification_token)
         return Response(serializer.data, status=status.HTTP_201_CREATED)
 
+    @staticmethod
+    def verification_link(email,token):
+        subject = "Activation for your account"
+        message = f'Please Activate your account below.\n{token}'
```

Comment:
@sebbss, the body of the email should reference a URL which can be clicked to activate the account.